Logger.lua

```diff
@@ -55,24 +51,19 @@ end
 Logger.Path       = ""
 Logger.Namespace  = "Logger"
 Logger.Console    = false
-Logger.DefaultLvl = 1
+Logger.DefaultLvl = 3
 
 Logger.Type = {
-  { Name = "TRACE", Color = "32" },
-  { Name = "DEBUG", Color = "36" },
-  { Name = "INFO.", Color = "34" },
-  { Name = "WARN.", Color = "33" },
-  { Name = "ERROR", Color = "31" },
-  { Name = "FATAL", Color = "35" },
-  { Name = "OTHER", Color = "30" }
+  { Name = "OTHER", Color = "30" }, { Name = "TRACE", Color = "32" },
```

Comment:
i formatted this part because it looks nicer like this, i also changed the level of importance, so 6+ is all "bad" things now

formatee esta parte porque se ve mejor asi, tambien cambie el orden de importancia asi 6+ es solo cosas "malas"

Reply:
Excelente, ya tenía planeado hacerlo de una manera ligeramente distinta (`OTHER` pasaría a ser 0), pero de esta forma también funciona